js/controller.js

```diff
@@ -44,6 +44,8 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       "isLiveStream": false,
       "screenToShow": null,
       "playerState": null,
+      "isPlaying": false,
```

Comment:
I think we already keep track of this with `this.state.playerState`

Reply:
`playerState` has a `CONSTANTS.STATE.PAUSE` state, as well as other states. I want to keep track of when we are actively showing a video, paused or playing. Is there something existing I can use? Should I rename this?

Reply:
Actually I'll try to use `playerState.` It might work out.

Reply:
Done.